docs/concepts/models.md

```diff
@@ -168,6 +171,53 @@ Models possess the following methods and attributes:
     See [Changes to `pydantic.BaseModel`](../migration.md#changes-to-pydanticbasemodel) in the
     [Migration Guide](../migration.md) for details on changes from Pydantic V1.
 
+## Data conversion
+
+Pydantic may cast input data to force it to conform to model field types,
+and in some cases this may result in a loss of information.
+For example:
+
+```python
+from pydantic import BaseModel
+
+
+class Model(BaseModel):
+    a: int
+    b: float
+    c: str
+
+
+print(Model(a=3.000, b='2.72', c=b'binary data').model_dump())
+#> {'a': 3, 'b': 2.72, 'c': 'binary data'}
```

Comment:
Well in this case I don't think there's any data loss per say, I mean it was converted but we haven't "lost" anything. Do you have an example where this can be the case?

Reply:
I guess you can just say you're losing precision for `a`?

Reply:
Yeah, I'm fine with leaving it however you prefer then. Fair point re data loss.